Sources/Protocols/MessagesLayoutDelegate.swift

```diff
@@ -145,9 +172,16 @@ public extension MessagesLayoutDelegate {
             return UIEdgeInsets(top: 0, left: 4, bottom: 0, right: 30)
         }
     }
-
-    func avatarAlignment(for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> AvatarAlignment {
-        return .cellBottom
+    
+    func avatarPosition(for message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> AvatarPosition {
+        guard let dataSource = messagesCollectionView.messagesDataSource else {
+            return AvatarPosition(horizontal: .cellLeading, vertical: .messageBottom)
+        }
```

Comment:
I want to eliminate all these else return cases in the library. Should be a `fatalError()`

Reply:
Works for me